src/main/java/com/github/krzysiekjodlowski/ox/FieldNumber.java

```diff
@@ -0,0 +1,18 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Represents each player move.
+ * @author Krzysztof Jodlowski
+ */
+class FieldNumber {
```

Comment:
What do you need this class for? Why not just declare a variable as an int?

Reply:
In Map i must keep Object, so i decided to create my own wrapper with name reflecting its usage.

Reply:
OK